apis/pipeline/BUILD

```diff
@@ -2,14 +2,14 @@ subinclude("///pleasings2//openapi")
 
 openapi_library(
     name = "pipeline",
-    spec = "pipeline.yaml",
-    generator = "go-server",
     out = ".gen/pipeline",
     additional_properties = {
         "packageName": "pipeline",
         "withGoCodegenComment": "true",
         "sourceFolder": "pipeline",
     },
+    generator = "go-server",
+    spec = "pipeline.yaml",
```

Comment:
Is this generated?
If not, why is the generator put here and not above the `name` field?

Reply:
It's auto formatted by wollemi using the [buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md) tool

Reply:
I'm a bit sad personally they didn't go for the lexicographical order, so close yet so far.